posthog/async_migrations/utils.py

```diff
@@ -12,10 +14,19 @@
 from posthog.email import is_email_available
 from posthog.models.async_migration import AsyncMigration, AsyncMigrationError, MigrationStatus
 from posthog.plugins.alert import AlertLevel, send_alert_to_plugins
+from posthog.utils import get_machine_id
 
 logger = structlog.get_logger(__name__)
 
 
+def send_analytics_to_posthog(event, data):
+    try:
+        posthoganalytics.project_api_key = "sTMFPsFhdP1Ssg"
+        posthoganalytics.capture(get_machine_id(), event, data, groups={"instance": settings.SITE_URL})
```

Comment:
can we send some other useful information? 

org name, team name, etc? you can take a look at how other parts of the codebase do this